public/styles/styles.css

```diff
@@ -1,10 +1,34 @@
 body {
   color: #333;
   font-family: Helvetica, Arial, sans-serif;
-  background-color: skyblue; /* Sanity Check! */
+  /*background-color: blue;*/
+  background-image: url("../images/friends.jpg");
```

Comment:
This is where I'd usually just write `/images/friends.jpg` since that's the path from the root.